content/cn/docs/clients/restful-api/indexlabel.md

```
@@ -6,14 +6,14 @@ weight: 5
 
 ### 1.5 IndexLabel
 
-假设已经创建好了1.1.3中的 PropertyKeys 、1.2.3中的 VertexLabels 以及 1.3.3中的 EdgeLabels
+假设已经创建好了 1.1.3 中的 PropertyKeys、1.2.3 中的 VertexLabels 以及 1.3.3 中的 EdgeLabels
 
-#### 1.5.1 创建一个IndexLabel
+#### 1.5.1 创建一个 IndexLabel
 
 ##### Method & Url
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)